doc/source/overview/high_level.md

```diff
@@ -0,0 +1,1084 @@
+# Introduction
+
+```{contents}
+:depth: 4
+:local: true
+```
+
+## What is Explainability?
+
+**Explainability provides us with algorithms that give insights into trained models predictions.** It allows us to
+answer questions such as:
+
+- How does a prediction **change** dependent on feature inputs?
+- What features **are** or **are not** important for a given prediction to hold?
+- What set of features would you have to minimally **change** to obtain a **new** prediction of your choosing?
+- How does each feature **contribute** to a model's prediction?
+
+```{image} images/exp-aug.png
+:align: center
+:alt: Model augmented with explainabilty 
+```
+
+Alibi provides a set of **algorithms** or **methods** known as **explainers**. Each explainer provides some kind of
+insight about a model. The set of insights available given a trained model is dependent on a number of factors. For
+instance, if the model is a [regression](https://en.wikipedia.org/wiki/Regression_analysis) it makes sense to ask how
+the prediction varies for some regressor. Whereas it doesn't make sense to ask what minimal change is required to
+obtain a new class prediction. In general, given a model the explainers we can use are constrained by:
```

Comment:
Mention that the explainers we can use are the ones in Alibi, as fundamentally explainers are not limited to e.g. regression or classification tasks.

Reply:
I don't think so, no. Otherwise you can end up with a pretty exhaustive list and there might also be some confusion about what could be explained and what type has practical explanation methods.

Reply:
Ok great